src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java

```diff
@@ -67,6 +67,9 @@ public class FilingsService {
   @Value("${FEATURE_FLAG_ENABLE_TRUSTS_CHIPS_1502023}")
   private boolean isTrustsSubmissionThroughWebEnabled;
 
+  @Value("${PUBLIC_API_IDENTITY_HASH_SALT}")
```

Comment:
Don't need to add this (yet) as no functional code that uses it is submitted.

Reply:
Ok, next PR :-(